### DISCUSSION TITLE
HVAC settings dialog box

### DISCUSSION THREAD

**Discussion by fidran** [firemodels/fds#12675] [General]
I can not find the path to the HVAC settings dialog box Figure 3.19 in SMV guide.
![HVAC dialog](https://github.com/firemodels/fds/assets/40716592/5d2066e1-72d3-42a0-96c2-9d1484528f56)

**Comment by gforney** [comment]
I'll add a note the to the user's guide to make this clearer.  Note this dialog box only appears for cases that have hvac networks
Dialogs>View>HVAC Settings
![hvac_dialog](https://github.com/firemodels/fds/assets/12403014/954984d2-4ffd-4b48-ac2e-1e92deb73eea)

**Reply by fidran** [reply]
HVAC settings will not show up if the results is from FDS older than 6.8.0. Figured that after a while ..
